docs/resources/watch.md

```diff
@@ -0,0 +1,151 @@
+---
```

Comment:
Is this the correct path?

Reply:
if we will figure out how to make `.SchemaMarkdown` work with resources (not only provider, to generate index.md), then no, it's not a correct path. 
Correct path for the generated doc will be `docs/resources/xray_watch/xray_watch.md`